src/components/Projects/WBS/WBSDetail/AddTask/AddTaskModal.jsx

```diff
@@ -10,7 +10,7 @@ import 'react-day-picker/lib/style.css';
 import dateFnsFormat from 'date-fns/format';
 import { Editor } from '@tinymce/tinymce-react';
 
-const AddTaskModal = (props) => {
+const AddTaskModal = props => {
```

Comment:
No functional changes in this file.

Reply:
Yess,I tried like 3 times remove the ESLint stuff that was made in this file, sorry.

Reply:
No worries!